module-apps/application-call/ApplicationCall.cpp

```diff
@@ -6,6 +6,7 @@
 #include "CallWindow.hpp"
```

Comment:
One case still needs to be fixed. Speaker Icon gets lost when Bluetooth headphones are connected before the call starts.